addons/optionsmenu/XEH_preInit.sqf

```diff
@@ -15,4 +15,55 @@ GVAR(serverSideValues) = [];
 GVAR(categories) = [];
 GVAR(currentCategorySelection) = 0;
 
+
+[] spawn {
+    sleep 0.1; //is3den command has problems at pre-init, need spawn/sleep
```

Comment:
A lot of things are done on postinit. Doesn't that mean all these are now broken in 3den?